src/tests/src/solver/simulation/CMakeLists.txt

```diff
@@ -92,14 +92,8 @@ set_property(TEST time_series PROPERTY LABELS unit)
 # ===================================
 # Tests on hydro final reservoir level functions
 # ===================================
-set(SRC_HYDRO_FINAL_LEVEL
-	${src_libs_antares_study}/include/antares/study/parts/hydro/finallevelinflowsmodifyer.h
-	${src_libs_antares_study}/parts/hydro/finallevelinflowsmodifyer.cpp
```

Comment:
Caused **finallevelinflowsmodifyer.cpp** to compile twice : one when compiling solver, once when compiling these unit tests